packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -523,6 +530,43 @@ export abstract class FluidDataStoreContext
 		return factory;
 	}
 
+	createChildDataStoreSync<T extends IFluidDataStoreFactory>(
```

Comment:
I'd remove "Sync" from the name. It should be obvious from API shape, and we do not have any other API on context today